### DISCUSSION TITLE
【新主题】 hugo-theme-yinyang

### DISCUSSION THREAD

**Discussion by spacding** [tangly1024/NotionNext#2002] [Ideas]
**为什么提出这个新的特性改动**
您好，最近浏览到一款非常简洁，个人非常喜欢的主题：https://github.com/joway/hugo-theme-yinyang 
示例网站：https://blog.joway.io/
但是该主题是基于Hugo的，由于我对此所知甚少，尽管做了一些尝试，但还是未能成功，所以请问作者是否可以考虑下移植该主题。

**描述一下你推荐的解决方案**
如果作者**有兴趣**并且**有时间**的话希望考虑下移植到本博客框架的主题上，当然，一切基于您的意愿，感谢！

**Comment by jarlin8** [comment]
很简洁的主题

**Comment by tangly1024** [comment]
很漂亮的主题，但是短期内没有精力移植这款主题；

也许另外有大神可以尝试做一下？
